WordPress/Classes/ViewRelated/NUX/WordPressAuthenticationManager.swift

```diff
@@ -575,7 +575,8 @@ private extension WordPressAuthenticationManager {
         // Otherwise, show the My Site screen for the specified blog and after a short delay,
         // trigger the Quick Start tour
         self.windowManager.showAppUI(for: blog, completion: {
-            QuickStartTourGuide.shared.setupWithDelay(for: blog)
+            let quickStartType: QuickStartType = FeatureFlag.quickStartForExistingUsers.enabled ? .existingSite : .newSite
```

Comment:
RE: Existing Site tasks shown for the **Site creation after login** flow 

This is where the `.existingSite` type is assigned

Reply:
Updated in 9f90704332d2e390151da3ce9be1a3fe8b5d9d4c 👍